.gitignore

```diff
@@ -1 +1,32 @@
-.vscode/
\ No newline at end of file
+# Prerequisites
+*.d
+
+# Compiled Object files
+*.slo
+*.lo
+*.o
+*.obj
+
+# Precompiled Headers
+*.gch
+*.pch
+
+# Compiled Dynamic libraries
+*.so
+*.dylib
+*.dll
+
+# Compiled Static libraries
+*.lai
+*.la
+*.a
+*.lib
+
+# Executables
+*.exe
+*.out
+*.app
+
+# Other
+.vscode/
+test
```

Comment:
is this just blocking the file name test?

Reply:
yes, test was being listed as an un-staged change by git

Reply:
but test is a .exe file right? so wouldn't it be blocked by *.exe?

Reply:
Yes.

Reply:
Originally, I added test before adding the .exe and didn't put the two together.